src/components/photos/PhotoGalleryView.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import { Pressable, StyleSheet, View, Text, Dimensions } from "react-native";
+import { Icon, Image } from "react-native-elements";
+import ImageView from "react-native-image-viewing";
+import { Colors } from "../../utils/Colors";
+import { Spacing } from "../../utils/Spacing";
+
+interface Props {
+    source: string;
```

Comment:
Might be a good idea to receive the entire `Photo` object instead of just a source string. That way if we add more things to the photo object to display we will be receiving it automatically.
```suggestion
    photo: Photo;
```